apps/src/componentLibrary/textField/TextField.tsx

```diff
@@ -21,59 +21,96 @@ export interface TextFieldProps extends AriaAttributes {
   value?: string;
   /** TextField label */
   label?: string;
+  /** TextField helper message */
+  helperMessage?: string;
+  /** TextField helper icon */
+  helperIcon?: FontAwesomeV6IconProps;
+  /** TextField placeholder */
+  placeholder?: string;
+  /** Is TextField readOnly */
+  readOnly?: boolean;
   /** Is TextField disabled */
   disabled?: boolean;
-  /** Is TextField indeterminate */
-  readonly?: boolean;
+  /** TextField Error */
   error?: {message: string; hasError: boolean};
```

Comment:
Thanks for spotting this!
While developing, I was thinking about pre defining possible `errorMessage` for the the rendered `TextField` before error has occured and toggle shopwing it by passing true/false `error.hasError`.
Now taking a look on it with a fresh eye, I agree it might be easier to use only `errorMessage` instead. (making this update now)